src/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Counter from './components/counter.js';
+import SearchForm from './components/searchform.js';
+import GenreSelect from './components/genreselect.js';
+import './index.css';
+import './css/header.css'
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedgenre: null, 
+    };
+  }
+
+  handleSearch = (query) => {
+    alert(`Searching for: ${query}`);
+  };
+
+  handleGenreSelect = (genre) => {
+    this.setState({ selectedgenre: genre });
+    alert(`Selected genre: ${genre}`);
+  };
+
+  render() {
+    const selectedgenre = this.state.selectedgenre;
+    return (
+      <div className="div-container">
+        <Counter initialValue={0} />
+        <SearchForm
+          initialQuery="What do you want to watch?"
+          onSearch={this.handleSearch}
+        />
+        <GenreSelect
+          genres={['All', 'Documentary', 'Comedy', 'Horror', 'Crime']}
+          selectedGenre={selectedgenre}
```

Comment:
Initial genre is not being displayed in UI. Please look into it.

Reply:
Done! commit id: https://github.com/johari02shubhi/netflixroulette/pull/1/commits/766d0bfb6414ddd6cb648f9e35b484a9d58fb32f

Reply:
lgtm